### PR TITLE
New: Support hamlit for haml

### PR DESCRIPTION
**Pull request by maximeg**
I use hamlit (https://github.com/k0kubun/hamlit) to render my haml templates.

I added support for it.

Feel free to tell me how to improve this PR :)

### CODE REVIEW

File: lib/gettext_i18n_rails/base_parser.rb [hunk 1]
```diff
@@ -15,18 +15,34 @@ def self.parse(file, _options = {}, msgids = [])
...
+          require library
+          true
+        rescue LoadError
+          false
+        end
+      end
+
+      unless loaded
+        if libraries.size == 1
+          puts "A #{extension} file was found, but #{libraries.first} library could not be found, so nothing will be parsed..."
```
Comment (grosser) [inline lib/gettext_i18n_rails/base_parser.rb]: let's just go with ugly english :D 

`no library could be found #{libraries.join(', )}`

File: lib/gettext_i18n_rails/base_parser.rb [hunk 2]
```diff
@@ -15,18 +15,34 @@ def self.parse(file, _options = {}, msgids = [])
...
+    end
+
     def self.load_library
       return true if @library_loaded
 
-      begin
-        require extension
-      rescue LoadError
-        puts "A #{extension} file was found, but #{extension} library could not be found, so nothing will be parsed..."
+      loaded = libraries.find do |library|
```
Comment (grosser) [inline lib/gettext_i18n_rails/base_parser.rb]: prefer detect ... find always makes me think activerecord :D

**Comment by grosser** [comment]
so this is to avoid having to add haml to your project or is there hamlit syntax that does not work under haml ?

**Comment by grosser** [comment]
tests ? :)

**Comment by maximeg** [comment]
Yes, this is not to have to add `haml` in the projects as it conflicts with `hamlit` (they both want to be the template handler for haml).

I'm not sure how to test it. I'll propose something :)

**Comment by maximeg** [comment]
Rails 4.2 and Ruby 1.9.3 made the install of hamlit fail (game of versions).

We could add this to the related gemfile:

    gem "hamlit", "0.2.0" if RUBY_VERSION =~ /\A1\.9/

But this would work only if hamlit is not in the lock files (which I didn't do yet).

**Comment by grosser** [comment]
merged via https://github.com/grosser/gettext_i18n_rails/pull/153

**Comment by grosser** [comment]
1.6.0

**Comment by maximeg** [comment]
Thanks :)

**Pull request merged**